tests/user/user_result_samples/extract_user_result_bad_attribute_error.json

```diff
@@ -8,12 +8,13 @@
         "errorFunction": 10,
         "errorCode": 2000,
         "errorReason": 68,
-        "errorMessage": "Invalid attribute value specified.",
+        "textInError": "name",
         "errorOffset": 149,
-        "textInError": "name"
+        "errorMessage": "Invalid attribute value specified."
```

Comment:
Is there a reason why the ordering was changed here?

Reply:
This is how you the error dictionary is/was built. This comes from IRRSMO00 rather than RACF, so this is coming from the "new" SecurityRequestError (previously NullResponseError) and this dictionary is truer to the error reported by SMO as this isn't associated with a "command", is not expected to have messages outside of one errorMessage, and uses a slightly different tag scheme for the same thing.

Reply:
Ok, so you just copied and pasted what pyRACF/IRRSMO00 gave you?